### DISCUSSION TITLE
LDCC: What to do about LD's admin rights.

### DISCUSSION THREAD

**Discussion by DavidEdwardDarden** [nss-day-cohort-47/C47-Discussions#68] [Help Me] [answered]
Should I leave LD alone or change it to say isAdmin: true... or does it matter?

In main.js.... under the //login/register listeners... where the values of userObject are being defined... I added isAdmin: false (as per the instructions)
It looks like this:
const userObject = {
			name: document.querySelector("input[name='registerName']").value,
			email: document.querySelector("input[name='registerEmail']").value,
			isAdmin: false
		}
This makes it so that when a new user registers they do not have admin rights... BUT in the dataset, LD is set to-> admin: true instead of isAdmin: true.

Should I leave LD alone or change it to say isAdmin: true... or does it matter?

**Comment by coltmay** [comment, marked as answer]
I changed the key name.  So long as the data matches for every user, I don't think the key name makes a difference.  I'm just making sure I'm accessing it correctly.

**Reply by AutumnFoto** [reply]
this thread helped me! thanks yall lol

**Comment by Chadwell-Clark** [comment]
I did not change the admin keys as it was not requested by the client. It might be good to contact the client and ask them whether they are aware of the different keys in the users section of the database. I agree that the keys being different does not seem to be a problem at this time.
